programs-service/src/services/programs.js

```diff
@@ -0,0 +1,104 @@
+let Program = require('@models/program');
+let {EntityNotFoundError, EntitySaveError} = require('@serviceErrors/');
+const {InvalidIdError} = require('@dataErrors/');
```

Comment:
data errors should not be used on the service layer.

Reply:
I didn't change the error mechanism nor have I used it, since you've told me once that you have a different idea in mind